### ISSUE TITLE
USA facts download is breaking

### ISSUE THREAD

**Issue by bolliger32** [bolliger32/gpl-covid#108]
We're getting one county with null imputed values, which is raising an error during one of @kendonB's checks (thanks for adding those!)

```
Browse[1]> usa_county_data_standardised[is.na(usa_county_data_standardised$cum_confirmed_cases_imputed),]
# A tibble: 3 x 13
  date       adm0_name adm1_name adm2_name population cum_confirmed_c… cum_confirmed_c… active_cases active_cases_im… cum_deaths
  <date>     <chr>     <chr>     <chr>          <dbl>            <dbl>            <dbl>        <dbl>            <dbl>      <dbl>
1 2020-04-01 USA       Wyoming   Weston          7208               NA               NA           NA               NA         NA
2 2020-04-02 USA       Wyoming   Weston          7208               NA               NA           NA               NA         NA
3 2020-04-03 USA       Wyoming   Weston          7208               NA               NA           NA               NA         NA
# … with 3 more variables: cum_deaths_imputed <dbl>, cum_recoveries <dbl>, cum_recoveries_imputed <dbl>
```

**Comment by bolliger32** [comment]
Closed by #110

**Comment by bolliger32** [comment]
Reopening for new usafacts issue:
```
Error in if (all((first_issue %>% pull({ : 
  missing value where TRUE/FALSE needed
```

**Comment by bolliger32** [comment]
Fixed by #117
